force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +180,9 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
-
+    Integer randomIntegerResult = Integer.valueOf((Math.random() * 51)+ 50);
         // Return the random integer
-        return null; // Replace null with the variable you used to store the result
+        return randomIntegerResult; // Replace null with the variable you used to store the result
```

Comment:
good job. But watch your formatting. Keeping code readable is essential.  Your first line needs to be indented.